Applications/WebPortal/WebApp/library/source/pages/account_page/risk_page/risk_page.tsx

```diff
@@ -0,0 +1,164 @@
+import * as Nexus from 'nexus';
+import * as React from 'react';
+import { DisplaySize, HLine, PageWrapper } from '../../..';
+import { SubmissionInput } from '..';
+import { RiskParametersView } from '.';
+
+interface Properties {
+
+  /** Used to lookup currency names and symbols. */
+  currencyDatabase: Nexus.CurrencyDatabase;
+
+  /** The type of display to render on. */
+  displaySize: DisplaySize;
+
+  /** The risk parameters to display. */
+  parameters: Nexus.RiskParameters;
+
+  /** The account's roles. */
+  roles: Nexus.AccountRoles;
+
+  /** Whether an error occurred. */
+  isError?: boolean;
+
+  /** Whether the submit button is enabled. */
```

Comment:
This should be determined by the ```roles```.

Reply:
How?

Reply:
What does this prop do? I might misunderstand it.

Reply:
Which prop are we talking about?

Reply:
The ```isSubmitEnabled```, when would it be or not be enabled?

Reply:
isSubmitEnabled?: boolean;

Reply:
It might not be enabled if no changes are made to the parameters page.

Alternatively it would make sense to just remove it entirely.

Reply:
Remove it if you can.